corehq/apps/cloudcare/static/cloudcare/css/formplayer.css

```diff
@@ -4,4 +4,23 @@
 
 .module-icon {
     max-height: 100px;
-}
\ No newline at end of file
+}
+
+body {
+    margin: 4px;
+}
+
+.grid-wrapper {
+    display: grid;
+    grid-template-columns: repeat(12, 60px) min-content 60px;
+    grid-template-rows: repeat(6, 60px) min-content 60px;
+    background-color: #fff;
+    color: #444;
```

Comment:
should use colors from HQ's palette: https://www.commcarehq.org/styleguide/colors/